client/src/common/components/Comments/Comment.scss

```diff
@@ -0,0 +1,39 @@
+.comment {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  box-shadow: 0 2px 10px rgba($shadow-color, 0.1);
+  padding: 16px;
+
+  &__body {
+    text-align: left;
+  }
+
+  &__author {
```

Comment:
I wanted to make it visible :P 
cause with font size 14px you would say it's too small referring to WCAG 

I agree it's too big. 
Same about your previous comment, I've decreased padding because this link-buttons are too big and ugly.
It's hard to choose between a good looking element or WCAAG compatible

Reply:
maybe the better would be to change color a little bit instead of changing the font-weight? Because now it meets only the requirements for AA anyway (not for AAA)?

So, we have to decide it the app should be pretty for use or should be easy to use for the users